src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, Color color) {
+        this.radius = radius;
+        super.color = color;
+    }

```

Comment:
According to the checklist, `super.color` should not be used. Instead, use the constructor in the superclass to set the color.